apps/site/assets/css/_schedule-page.scss

```diff
@@ -34,3 +34,56 @@
 .m-schedule-page__sidebar-list {
   padding-left: $base-spacing * 1.5;
 }
+
+.schedule-finder {
+  background-color: $gray-bordered-background;
+  padding: $base-spacing;
+
+  .c-modal__trigger-wrapper {
+    display: inline; // otherwise clicking next to the button fires the event
+  }
+}
+
+.schedule-finder__heading {
+  color: $brand-primary;
+  margin-top: 0;
+
+  &::before {
+    content: '\f073'; // font-awesome calendar
+    font-family: FontAwesome;
+    font-weight: normal;
+    margin-right: $base-spacing / 3;
+  }
+}
+
+.schedule-finder__select-container {
+  background-color: $white;
+  border: solid $border-width $brand-primary-dark;
+  border-radius: $border-radius;
+  margin: $base-spacing / 2 0;
+  padding: $base-spacing / 2;
+  position: relative;
+
+  &.error {
+    border: solid 2px $error-text;
+  }
+}
+
+.schedule-finder__select {
+  appearance: none;
+  background-color: $white;
+  border: 0;
+  width: 100%;
+
+  @include media-breakpoint-only(md) {
+    width: 90%;
+  }
+}
+
+.schedule-finder__arrow {
```

Comment:
It'd be great for this to be an SVG. If you think it's out of scope, we can add it as a polish ticket.

@ingridpierre can we add the task of providing an SVG to your queue too? We can then replace it on the accordions too.

Reply:
I will add the SVG -- at the time I did this, I didn't have the SVG

Reply:
SVG: https://drive.google.com/file/d/1hQTGZyOM5uP1RA3QUkLkFipr7hh_KHAv/view?usp=sharing